internals/core_models.py

```diff
@@ -954,3 +954,206 @@ def put(self, notify=True, **kwargs):
+
+  # Metadata: Process information
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+
+  # Implementation in Chrome
+  impl_status_chrome = ndb.IntegerProperty(required=True)
+  flag_name = ndb.StringProperty()
+  ongoing_constraints = ndb.TextProperty()
+
+  # Gate: Adoption
+  motivation = ndb.TextProperty()
+  devtrial_instructions = ndb.TextProperty()
+  activation_risks = ndb.TextProperty()
+  measurement = ndb.TextProperty()
+
+  # Gate: Standardization & Interop
+  initial_public_proposal_url = ndb.StringProperty()
+  explainer_links = ndb.StringProperty(repeated=True)
+  requires_embedder_support = ndb.BooleanProperty(default=False)
+  standard_maturity = ndb.IntegerProperty(required=True, default=UNSET_STD)
+  spec_link = ndb.StringProperty()
+  api_spec = ndb.BooleanProperty(default=False)
+  spec_mentors = ndb.StringProperty(repeated=True)
+  interop_compat_risks = ndb.TextProperty()
+  prefixed = ndb.BooleanProperty()
+  all_platforms = ndb.BooleanProperty()
+  all_platforms_descr = ndb.TextProperty()
+  tag_review = ndb.StringProperty()
+  tag_review_status = ndb.IntegerProperty(default=REVIEW_PENDING)
+  non_oss_deps = ndb.TextProperty()
+  anticipated_spec_changes = ndb.TextProperty()
+
+  ff_views = ndb.IntegerProperty(required=True, default=NO_PUBLIC_SIGNALS)
+  safari_views = ndb.IntegerProperty(required=True, default=NO_PUBLIC_SIGNALS)
+  web_dev_views = ndb.IntegerProperty(required=True)
+  ff_views_link = ndb.StringProperty()
+  safari_views_link = ndb.StringProperty()
+  web_dev_views_link = ndb.StringProperty()
+  ff_views_notes = ndb.StringProperty()
+  safari_views_notes = ndb.TextProperty()
+  web_dev_views_notes = ndb.TextProperty()
+  other_views_notes = ndb.TextProperty()
+
+  # Gate: Security & Privacy
+  security_risks = ndb.TextProperty()
+  security_review_status = ndb.IntegerProperty(default=REVIEW_PENDING)
+  privacy_review_status = ndb.IntegerProperty(default=REVIEW_PENDING)
+
+  # Gate: Testing / Regressions
+  ergonomics_risks = ndb.TextProperty()
+  wpt = ndb.BooleanProperty()
+  wpt_descr = ndb.TextProperty()
+  webview_risks = ndb.TextProperty()
+
+  # Gate: Devrel & Docs
+  devrel = ndb.StringProperty(repeated=True)
+  debuggability = ndb.TextProperty()
+  doc_links = ndb.StringProperty(repeated=True)
+  sample_links = ndb.StringProperty(repeated=True)
+
+  DEFAULT_CACHE_KEY = 'FeatureEntries'
+
+  def __init__(self, *args, **kwargs):
+    # Initialise Feature.blink_components with a default value.  If
+    # name is present in kwargs then it would mean constructor is
+    # being called for creating a new feature rather than for fetching
+    # an existing feature.
+    if 'name' in kwargs:
+      if 'blink_components' not in kwargs:
+        kwargs['blink_components'] = [BlinkComponent.DEFAULT_COMPONENT]
```

Comment:
BlinkComponent is defined in user_models. I see an undefined variable error on my lint

Reply:
Thanks for catching this.
I sent a follow-up PR: https://github.com/GoogleChrome/chromium-dashboard/pull/new/20220831-fix-reference-to-DEFAULT_COMPONENT